### DISCUSSION TITLE
/language/cpp/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#219] [Announcements]
# /language/cpp/

👩🏻‍💻🧑🏻‍💻 既然知道学校教得一般，那为什么还要跟着学校的课程安排呢？

https://cs-college-1gjpkwo5adcfc76a-1258913748.tcloudbaseapp.com/language/cpp/

**Comment by tomstillcoding** [comment]
测试

**Comment by Mister-Hope** [comment]
Use your own repo.
